pkg/sqlcmd/sqlcmd.go

```diff
@@ -44,6 +44,8 @@ type Console interface {
 	ReadPassword(prompt string) ([]byte, error)
 	// SetPrompt sets the prompt text shown to input the next line
 	SetPrompt(s string)
+	// Close clears any buffers and closes open file handles
+	Close()

```

Comment:
I'm not seeing an implementation for this method, is this by design?

Reply:
it's already implemented. https://github.com/microsoft/go-sqlcmd/blob/e3972914801bc3492b49b6fe642a7e931e088be3/pkg/console/console.go#L39